os/drivers/audio/alc5658.h

```diff
@@ -16,11 +16,19 @@
  *
  ****************************************************************************/
 /****************************************************************************
- * drivers/audio/alc5658.h
+ * drivers/audio/alc5658.c
```

Comment:
I think because of too many commits, you confused.
please change alc5658.c -> alc5658.h